phpstan-baseline.neon

```diff
@@ -32,4 +22,7 @@ parameters:
 			count: 1
 			path: src/Functions/NoNullableReturnTypeDeclarationRule.php
 
-
+		-
+			message: "#^Parameter \\#1 \\$argument of class ReflectionClass constructor expects class-string<T of object>\\|T of object, string given\\.$#"
```

Comment:
I didn't know how to fix this. 

I tried adding `/** @var class-string $className */` to the anonymous function, but that didn't help 🤷

Reply:
@rpkamp 

No problem, that is what the baseline is for. I do not think its worthwhile to fix every single issue reported by static code analysis.